src/Serialization/PlutusData.js

```diff
@@ -0,0 +1,33 @@
+/* global require exports BROWSER_RUNTIME */
+
+var lib;
+if (typeof BROWSER_RUNTIME != 'undefined' && BROWSER_RUNTIME) {
+    lib = require('@ngua/cardano-serialization-lib-browser');
+} else {
+    lib = require('@ngua/cardano-serialization-lib-nodejs');
+}
+
+exports._mkPlutusData_bytes = bytes => lib.PlutusData.new_bytes(bytes);
+exports._mkPlutusData_list = list => lib.PlutusData.new_list(list);
+exports._mkPlutusData_map = list => lib.PlutusData.new_map(list);
+exports._mkPlutusData_integer = int => lib.PlutusData.new_integer(int);
+exports._mkPlutusData_constr = constr => lib.PlutusData.new_constr_plutus_data(constr);
+
+exports._packPlutusList = helper => elems => helper.pack(lib.PlutusList, elems);
```

Comment:
the rest of the codebase uses maybe as a name for this helper, it feels more readable that way, don't you agree?